test-it/ViaTests/ScanFromLongString.via

```diff
@@ -0,0 +1,284 @@
+// Autogenerated Vireo assembly file
```

Comment:
WebVI is basically calling the following code:

![Remaining](https://user-images.githubusercontent.com/8452083/61154915-1cc1a780-a4b5-11e9-9bb5-f4d5075e4491.png)

![NotRemaining](https://user-images.githubusercontent.com/8452083/61154921-22b78880-a4b5-11e9-9790-7b37ed2bee7b.png)